reference/configuration.rst

```diff
@@ -213,16 +220,15 @@ The supported values for this enum are:
 
 * **debug:** Logs statement at DEBUG severity level.
 
-* **log:** Logs statement at LOG severity level.
+* **log:** Logs statement at LOG severity level. Log severity level will contain both the error, followed by :code:`STATEMENT` which contains the SQL query that was run.
```

Comment:
I'm not sure I understand the added comment here. Is there special behavior for `multi_task_query_log_level=log` not shared by the other levels?

Reply:
Log includes the statement following it, warning it is suppressed by default on cloud.